spec/support/publishing_api_helpers.rb

```diff
@@ -1,11 +1,12 @@
 module PublishingApiHelpers
   def write_payload(document)
-    document.delete("last_edited_at")
-    document.delete("publication_state")
-    document.delete("first_published_at")
-    document.delete("public_updated_at")
-    document.delete("state_history")
-    document
+    copy = FactoryGirl.create(document["document_type"], document)
```

Comment:
If the test makes changes to the document's attributes before it's passed to this method, they'll be lost because it's generating a new set of attributes from the factory.

I wonder if there's a way we can still create a deep copy of the hash without calling the factory. Something like `#deep_clone` ? I think Rails has methods to help with this.